tools/scripts/ffi.toml

```diff
@@ -147,6 +147,7 @@ args = ["build", "--package", "icu", "--target", "thumbv7m-none-eabi"]
 [tasks.diplomat-get-rev]
 description = "Get current Diplomat revision"
 category = "ICU4X Development"
+install_crate = { crate_name = "cargo-make", binary = "cargo", test_arg = ["make", "--version"], min_version = "0.35.0" }
```

Comment:
should this instead be a separate task that a couple tasks may depend on

Reply:
Done. Yep, agree with both your and Shane's reasonings.